setup.cfg

```diff
@@ -78,6 +78,14 @@ docs =
     scipy
     scikit-image
 
+[options.package_data]
+* = data/*, data/*/*, data/*/*/*, data/*/*/*/*, data/*/*/*/*/*, data/*/*/*/*/*/*
```

Comment:
Who are you so wise in the way of science?!

Reply:
🦆